mobile/library/java/io/envoyproxy/envoymobile/engine/EnvoyEngineImpl.java

```diff
@@ -65,18 +65,14 @@ public String dumpStats() {
   }
 
   /**
-   * Run the Envoy engine with the provided yaml string and log level.
-   *
-   * The envoyConfiguration is used to resolve the configurationYAML.
+   * Performs various JNI registration prior to engine running.
    *
    * @param configurationYAML The configuration yaml with which to start Envoy.
```

Comment:
nit: Can this @param and the logLevel @param now be removed.

Reply:
yes thanks